assets/web/js/cpdLightHc.js

```diff
@@ -56,18 +56,18 @@ export const cpdFontLetterSpacingHeadingSm = "-0.0166em";
 export const cpdFontLetterSpacingHeadingMd = "-0.0195em";
 export const cpdFontLetterSpacingHeadingLg = "-0.0209em";
 export const cpdFontLetterSpacingHeadingXl = "-0.0216em";
-export const cpdFontBodyXsRegular = "400 0.6875rem/1.5 Inter";
-export const cpdFontBodyXsMedium = "500 0.6875rem/1.5 Inter";
-export const cpdFontBodyXsSemibold = "600 0.6875rem/1.5 Inter";
-export const cpdFontBodySmRegular = "400 0.8125rem/1.5 Inter";
-export const cpdFontBodySmMedium = "500 0.8125rem/1.5 Inter";
-export const cpdFontBodySmSemibold = "600 0.8125rem/1.5 Inter";
-export const cpdFontBodyMdRegular = "400 0.9375rem/1.5 Inter";
-export const cpdFontBodyMdMedium = "500 0.9375rem/1.5 Inter";
-export const cpdFontBodyMdSemibold = "600 0.9375rem/1.5 Inter";
-export const cpdFontBodyLgRegular = "400 1.0625rem/1.5 Inter";
-export const cpdFontBodyLgMedium = "500 1.0625rem/1.5 Inter";
-export const cpdFontBodyLgSemibold = "600 1.0625rem/1.5 Inter";
+export const cpdFontBodyXsRegular = "400 0.688rem/1.5 Inter";
+export const cpdFontBodyXsMedium = "500 0.688rem/1.5 Inter";
+export const cpdFontBodyXsSemibold = "600 0.688rem/1.5 Inter";
+export const cpdFontBodySmRegular = "400 0.813rem/1.5 Inter";
+export const cpdFontBodySmMedium = "500 0.813rem/1.5 Inter";
+export const cpdFontBodySmSemibold = "600 0.813rem/1.5 Inter";
+export const cpdFontBodyMdRegular = "400 0.938rem/1.5 Inter";
+export const cpdFontBodyMdMedium = "500 0.938rem/1.5 Inter";
+export const cpdFontBodyMdSemibold = "600 0.938rem/1.5 Inter";
+export const cpdFontBodyLgRegular = "400 1.063rem/1.5 Inter";
+export const cpdFontBodyLgMedium = "500 1.063rem/1.5 Inter";
+export const cpdFontBodyLgSemibold = "600 1.063rem/1.5 Inter";
```

Comment:
These actually change the size, even if not by much. Is that ok?

Reply:
Oh, this should actually go into https://github.com/element-hq/compound-design-tokens/pull/87 😅

Reply:
Ah, I missed this before I merged the other PR. I'll try to research why Style Dictionary is rounding the values, and whether it messes with anything.

Reply:
(as in, I'll do that in another PR now that it's already too late…)